src/popup/components/Modals/PrivateKeyExport.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <Modal
+    class="private-key-export"
+    from-bottom
+    has-close-button
+    centered
+    @close="resolve"
+  >
+    <h1 class="text-heading-4 heading">
+      {{ $t('modals.privateKeyExport.title') }}
+    </h1>
```

Comment:
Following change wille render the text as `<h1>Some text</h1>` instead of `<h1> Some text </h1>`:
```suggestion
    <h1
      v-text="$t('modals.privateKeyExport.title')"
      class="text-heading-4 heading"
    >
```

Not a big thing, but in some cases helps to get rid of spacing issues.